oss/golang-gin/Makefile

```diff
@@ -0,0 +1,30 @@
+GO ?= go
+
+# Install the required dependencies
+install.deps:
+	@echo "==> Install dep"
+	@$(GO) get -u github.com/golang/dep/cmd/dep
+	@echo "==> Install dependencies"
+	@dep ensure
+.PHONY: install.deps
+
+# Install all the dependencies
+deps:
+	@echo "==> Install dependencies"
+	@dep ensure
+.PHONY: deps
+
+# Build all files.
+build:
+	@$(GO) build -o server ./...
+.PHONY: build
+
+# Build so it can be deployed by up.
+build.up:
+	@GOOS=linux GOARCH=amd64 $(GO) build -o server ./...
```

Comment:
everything looks good to me! The only thing I'd maybe change, is removing the Makefile since a lot of Go people tend to avoid Makefiles (I still like them personally).

What we don't have an example of though, is placing the Go cmd in say `./cmd/api/main.go` and using a custom build command to output `./server` in that case, I think that'd be helpful!

Reply:
@tj Do you mean moving `pong.go` to something like `./api/main.go` or something more.